samples/apps/copilot-chat-app/webapp/src/components/chat/ChatResourceList.tsx

```diff
@@ -0,0 +1,104 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+import {
+    Table,
+    TableBody,
+    TableCell,
+    TableCellLayout,
+    TableHeader,
+    TableHeaderCell,
+    TableRow,
+    makeStyles,
+    shorthands,
+    tokens,
+} from '@fluentui/react-components';
+import { DocumentPdfRegular, DocumentTextRegular } from '@fluentui/react-icons';
+import * as React from 'react';
+import { ChatMemorySource } from '../../libs/models/ChatMemorySource';
+import { useChat } from '../../libs/useChat';
+
+const useClasses = makeStyles({
+    root: {
+        ...shorthands.margin(tokens.spacingVerticalM, tokens.spacingHorizontalM),
+    },
+    table: {
+        backgroundColor: tokens.colorNeutralBackground1,
+    },
+    tableHeader: {
+        fontWeight: '600',
+    },
+});
+
+interface ChatResourceListProps {
+    chatId: string;
+}
+
+export const ChatResourceList: React.FC<ChatResourceListProps> = ({ chatId }) => {
+    const classes = useClasses();
+    const chat = useChat();
+    const [resources, setResources] = React.useState<ChatMemorySource[]>([]);
+
+    React.useEffect(() => {
+        chat.getChatMemorySources(chatId).then((sources) => {
+            setResources(sources);
+        });
+        // eslint-disable-next-line react-hooks/exhaustive-deps
+    }, [chatId]);
+
+    const items = resources.map((item) => ({
+        name: {
+            label: item.name,
+            icon: getFileIconByFileExtension(item.name),
+            url: item.hyperlink,
+        },
+        createdOn: {
+            label: new Date(item.createdOn).toLocaleTimeString([], {
+                hour: '2-digit',
+                minute: '2-digit',
+            }),
+            timestamp: item.createdOn,
+        },
+        sharedBy: { label: item.sharedBy, status: 'available' },
```

Comment:
Persona isn't needed here, just need a label with text only

Reply:
not needed anymore since I removed this column for now until we can properly fetch for the name instead of storing it